resources/lang/id/beatmaps.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'discussion-posts' => [
+        'store' => [
+            'error' => 'Gagal menyimpan post',
+        ],
+    ],
+
+    'discussion-votes' => [
+        'update' => [
+            'error' => 'Gagal memperbarui pilihan',
+        ],
+    ],
+
+    'discussions' => [
+        'allow_kudosu' => 'izinkan kudosu',
+        'delete' => 'hapus',
+        'deleted' => 'Dihapus oleh :editor :delete_time.',
+        'deny_kudosu' => 'tolak kudosu',
+        'edit' => 'sunting',
+        'edited' => 'Terakhir disunting oleh :editor :update_time.',
+        'kudosu_denied' => 'Perolehan kudosu dibatalkan.',
```

Comment:
"ditolak" is more suitable here, since 'deny_kudosu' variable translates to 'tolak kudosu'

Reply:
Will be applied